.travis.yml

```diff
@@ -0,0 +1,13 @@
+dist: trusty
+language: python
+script:
+  - ansible-lint ansible/setup.yml
+deploy:
+  - provider: script
+    script: deploy.sh
```

Comment:
If CI kicks in (e.g. when this PR is merged - if Travis is enabled on this repo) I expect this job to fail.

Please add:
* One line comment on what this script is meant to do;
* How this script integrate in the Travis CI flow. It looks like that if master of this repo is changed e.g. you add a developer to the list, then the CI is meant to kick in and update the configuration of the (already existing) hosts.

Reply:
I added more info in README in efa9504c7fd927de7c20664571ab09b6fe037207. I think it's better place to write about workflow instead in travis config.